src/strategies/dfyn-lp/index.ts

```diff
@@ -0,0 +1,81 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+const DFYN_SUBGRAPH_URL = {
+  '137': 'https://api.thegraph.com/subgraphs/name/ss-sonic/dfyn-v5'
+};
+
+export const author = 'vfatouros';
```

Comment:
```suggestion
export const author = 'vatsalgupta13';
```

Reply:
So basically its actually based on staked-uniswap strategy. But they hardcoded the subgraph url so I had to change the code.

Reply:
> So basically its actually based on staked-uniswap strategy. But they hardcoded the subgraph url so I had to change the code.

Ya I can understand that hardcoding is not good, but i see this is minor change so we can always change the strategy and pass optional params as per our requirement 

example:
```
await subgraphRequest(	
    options.subGraphURL ? options.subGraphURL : INFINITYPROTOCOL_SUBGRAPH_URL[network],	
    params
```

```
if(options.scoreMultiplier) {
  score[userAddress] = score[userAddress] * options.scoreMultiplier
}
```
and also would be great if we can document these options inside README.md

Reply:
Sure we can do that. Should I make the changes in staked-uniswap strategy or infinityprotocol-liquidity-pools strategy, both of them are actually the same with different subgraph requests.

Reply:
Any strategy should be fine as long as the previous strategy works well and your changes are small, optional, and well documented (doesn't disturb existing voting power calculation) make sure to add some test addresses into examples.json that returns a positive score with your changes

Reply:
Sure. Will do that.